### ISSUE TITLE
Support for built-in render pipeline?

### ISSUE THREAD

**Issue by elfin8er** [freezy/VisualPinball.Engine#467]
I am trying to test a table using Unity's built-in render pipeline, but balls don't seem to be working currently. According to [the documentation](https://docs.visualpinball.org/creators-guide/setup/installing-vpe.html): "the built-in renderer is legacy not recommended", but I see no mention if it not being supported.

**Unity version:** 2022.3.6f1

**Steps to reproduce:**
- Create a new Unity project with the 3D template
- Install VPE packages:
  - org.visualpinball.engine.unity
  - org.visualpinball.unity.assetlibrary
  - org.visualpinball.engine.pinmame
- Save a default .vpx table from VPX
- Import .vpx into VPE using Visual Pinball > Editor > Import Wizard
- Enter Play mode
- Pressing/holding Enter doesn't launch a ball

**Comment by arthurkehrwald** [comment]
I don't think it makes sense to support it. The built-in render pipeline is slowly being phased out and supporting three different render pipelines is just a maintenance nightmare. I can't think of any benefits for users either.

**Comment by freezy** [comment]
Yes, I think we'll remove support for the built-in renderer.

I think the path forward is to focus on HDRP and when Unity merges the pipelines in v7, we'll adapt accordingly.
